doc/input-output-reference/src/overview/group-system-availability-managers.tex

```diff
@@ -628,6 +637,16 @@ \subsubsection{Outputs}\label{outputs-9-008}
 
 HVAC,Average,Availability Manager Hybrid Ventilation Control Status {[]}
 
+HVAC,Average,Hybrid Ventilation Control HVAC System Opration Elapsed Time {[min]}
+
+HVAC,Average,Hybrid Ventilation Control Natual Ventilation Elapsed Time {[min]}
+
+HVAC,Average,Hybrid Ventilation Operative Temperature {[C]}
```

Comment:
It would be similar to Hybrid Ventilation Operative Temperature which duplicates Zone Operative Temperature.  But it's fine as-is.  This could be added later if requested.